src/Helper/IrisHelper.php

```diff
@@ -3,6 +3,7 @@
 namespace ElevenMiles\Iris\Helper;
 
 define('IRIS_ASSET_PATH', '/vendor/eleven-miles/iris/src/assets');
+define('WP_API_NAMESPACE', 'tilda/v1');
```

Comment:
do you think this could be inherited from a project's .env or perhaps default it to `iris/v1` if not overridden? Just thinking that it's then not tied to one project as such?